tests/longhaul/block_on_k8s_test.go

```diff
@@ -196,18 +203,21 @@ func (s *K8sBlockLongHaulSuite) dbOperation(i int) {
 	s.db.InsertRandomData()
 	s.db.InsertRandomData()
 	s.db.InsertRandomData()
+	s.db.SelectRandomData(5)
 	s.db.InsertRandomData()
 	s.db.InsertRandomData()
 	s.db.InsertRandomData()
+	s.db.SelectRandomData(10)
 
 	//delete Data
 	s.db.DeleteRandomRow()
+	s.db.SelectRandomData(20)
 
 }
 func (s *K8sBlockLongHaulSuite) TearDownSuite() {
 	s.db.CloseConnection()
-	s.kh.ResourceOperation("delete", s.mysqlAppPath)
-	s.storageClassOperation("mysql-pool", "delete")
+	//s.kh.ResourceOperation("delete", s.mysqlAppPath)
```

Comment:
why not clean these up?

Reply:
We want to repeat the test with the same setup, reuse the same volume basically.

Reply:
what test reuses them? This means another test has a dependency on this one?

Reply:
For the long-haul test, we don't want to delete the volume after each run, we want to set up a volume and use it for multiple hours or days. The tests deploy rook under a unique namespace, so other regression tests should not be affected. Also, we have called out that long-haul test doesn't clean up the test data in the readme.

Reply:
ok, would be great to add a comment here about why they aren't cleaned up. And if there is no intention of uncommenting these lines, seems like you could delete them.

Reply:
Added a comment to the test calling out no clean up